client/src/shared/components/article-preview.js

```diff
@@ -62,9 +62,9 @@ const ArticlePreview = props => {
         <p>{article.description}</p>
         <span>Read more...</span>
         <ul className="tag-list">
-          {article.tagList.map(tag => {
+          {article.tagList.map((tag, i) => {
```

Comment:
Actually, instead of using the index as the key, could we filter out duplicates from this list? It's supposed to be an aggregation of all the tags from all the blogposts and we shouldn't see duplicates here :)